test/kyberReserve.js

```diff
@@ -204,8 +204,10 @@ contract('KyberReserve', function(accounts) {
         //verify base rate
         let buyRate = await reserveInst.getConversionRate(ethAddress, tokenAdd[tokenInd], amountWei, currentBlock);
         let expectedRate = (new BigNumber(baseBuyRate[tokenInd]));
-        let destQty = (new BigNumber(amountWei).mul(baseBuyRate[tokenInd])).div(precisionUnits);
-        let extraBps = getExtraBpsForBuyQuantity(destQty);
+        let extraBps = compactBuyArr[tokenInd] * 10;
+        expectedRate = addBps(expectedRate, extraBps);
```

Comment:
why changes here?

Reply:
Incorrect calculation when adding bps, IIRC.

Reply:
Anyway, I think Mike fixed it in his branch. Merging his stuff.